### PR TITLE
add tgc/cci.20170813

### PR DESCRIPTION
**Pull request by SpaceIm**
Specify library name and version:  **lib/1.0**

This is also a good place to share with all of us **why you are submitting this PR** (specially if it is a new addition to ConanCenter): is it a dependency of other libraries you want to package? Are you the author of the library? Thanks!

---

- [ ] I've read the [guidelines](https://github.com/conan-io/conan-center-index/blob/master/docs/how_to_add_packages.md) for contributing.
- [ ] I've followed the [PEP8](https://www.python.org/dev/peps/pep-0008/) style guides for Python code in the recipes.
- [ ] I've used the [latest](https://github.com/conan-io/conan/releases/latest) Conan client version.
- [ ] I've tried at least one configuration locally with the
      [conan-center hook](https://github.com/conan-io/hooks.git) activated.

### CODE REVIEW

**Review by prince-chrismc — APPROVED** [review]

**Review by jgsogo — APPROVED** [review]

**Review by SSE4 — APPROVED** [review]

**Review by madebr — APPROVED** [review]

File: recipes/tgc/all/test_package/test_package.c [hunk 1]
```diff
@@ -0,0 +1,18 @@
+#include <tgc.h>
+
+static tgc_t gc;
+
+static void example_function() {
+    char *message = tgc_alloc(&gc, 64);
+    strcpy(message, "No More Memory Leaks!");
```
Comment (madebr) [inline recipes/tgc/all/test_package/test_package.c:7]: `"tgc.h"` probably includes it already, but you need to include the `string.h` header for strcpy.

Comment (madebr) [inline recipes/tgc/all/test_package/test_package.c:7]: `"tgc.h"` probably includes it already, but you need to include the `string.h` header for strcpy.

Comment (SpaceIm) [reply recipes/tgc/all/test_package/test_package.c:7]: Correct but I can't edit right now.

Comment (madebr) [reply recipes/tgc/all/test_package/test_package.c:7]: Then don't worry about it!
For now :smile:

**Pull request merged**